extensions/objects/templates/filepath-editor.directive.ts

```diff
@@ -794,17 +794,8 @@ angular.module('oppia').directive('filepathEditor', [
 
         ctrl.generateImageFilename = function(height, width,
             imageType = 'png') {
-          var date = new Date();
-          return 'img_' +
-            date.getFullYear() +
-            ('0' + (date.getMonth() + 1)).slice(-2) +
-            ('0' + date.getDate()).slice(-2) +
-            '_' +
-            ('0' + date.getHours()).slice(-2) +
-            ('0' + date.getMinutes()).slice(-2) +
-            ('0' + date.getSeconds()).slice(-2) +
-            '_' +
-            Math.random().toString(36).substr(2, 10) +
+          var imageFilename = ImageUploadHelperService.generateImageFilename();
```

Comment:
Hm, one thought. Do you think we should add height/width information to the topic thumbnail image too, similar to what we do for the images in lessons?

The reason we originally did that was because, if an image takes a while to load, then knowing its intended dimensions means that we can avoid the page juddering once it loads. It seems to me like this might be the case for all images.

Reply:
Done. Used 150x150 now for topic thumbnail but this will be changed when we allow the cropping tool to crop at 16:9 aspect ratio.